flytekit/core/resources.py

```diff
@@ -96,3 +100,48 @@ def convert_resources_to_resource_model(
     if limits is not None:
         limit_entries = _convert_resources_to_resource_entries(limits)
     return task_models.Resources(requests=request_entries, limits=limit_entries)
+
+
+def construct_k8s_pod_spec_from_resources(
+    k8s_pod_name: str,
+    requests: Optional[Resources],
+    limits: Optional[Resources],
+) -> dict[str, Any]:
+    def construct_k8s_pods_resources(resources: Optional[Resources]):
```

Comment:
can you add the prefix `_` to this function? It's already internal to this function, it's more of a convention.